changelog/issue-3148.md

```diff
@@ -0,0 +1,6 @@
+audience: deployers
+level: major
```

Comment:
I think this can be minor, since it does not require a TCW

Reply:
👍